### ISSUE TITLE
Add something to support parse options from shell script command line 

### ISSUE THREAD

**Issue by qinshulei** [tdenniston/bish#52]
Hi @tdenniston ,

 I think it's important for script to support parse options from shell script command line .But it seems that there is no good way to do this in bish. 
 so I hope you can add something to support this.

Thank you.
shulei

**Comment by tdenniston** [comment]
@qinshulei This depends on #53, which should hopefully be done soon (I'm working on it now).

**Comment by tdenniston** [comment]
@qinshulei You should be able to use the built in 'args' array now for this task. For an example, see the `tests/args.bish` file. If there are any problems, open a new issue for them. Thanks!
